src/amoc_api_scenario_handler.erl

```diff
@@ -81,8 +81,8 @@ resource_exists(Req, State = #state{resource = Resource}) ->
 -spec to_json(cowboy_req:req(), state()) -> 
     {jsx:json_text(), cowboy_req:req(), state()}.
 to_json(Req0, State = #state{resource = Resource}) ->
-    %% Need to retrive state of module here
-    Reply = jsx:encode([{module, Resource}]),
+    Status = amoc_controller:test_status(erlang:list_to_atom(Resource)),
```

Comment:
What are possible test status values? It seems that `amoc_controller:test_status/1` returns true or false. It's not logical to have test status of true or false regarding a test scenario.

As far as I remember a test can be either running or loaded from the Amoc perspective. And it would be nice if these statuses have a common set with those defined on the Tide side. @kklocek and @arkgil can you discuss that together an describe possible status values somewhere? Then @kklocek please adjust this PR.

Additionally I want to have this status values described within the spec by something like:

```erlang

-type scenario_status() :: loaded | running.

 -spec test_status(scenario_name()) -> scenario_status().
test_status(ScenarioName) ->
    {ok, Res} = gen_server:call(?SERVER, {status, ScenarioName}),
    Res.
```